distgen/version.py

```diff
@@ -1 +0,0 @@
-dg_version = "1.18"
```

Comment:
I think we can afford doing this module API change.  :shrug: I wish I knew how to do this properly

Reply:
Well, it is unfortunate that it seems there is no way to do this properly yet. Like the python module has metadata from `pyproject.toml` which contains the version, that should be the primary source of version, but it also complicates how to fetch that version in other places. (Like in the Makefile)

I would love to have the `pyproject.toml` version primary source, but also I wouldn't want to keep the version tracked in multiple places.